### DISCUSSION TITLE
Symmetry about an arbitrary line

### DISCUSSION THREAD

**Discussion by ndaman** [idaholab/moose#20471] [Q&A General] [answered]
I'm trying to impose a symmetric boundary about a diagonal line in my model (such that the vector variable,  u, dotted with the normal vector to that line, n = 0). I did find the penalty inclined no displacement (https://mooseframework.inl.gov/source/bcs/PenaltyInclinedNoDisplacementBC.html), is that the best way to reach this boundary condition?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Is this line inside the mesh? Or on the edge?
The regular BC, no flux, is usually pretty good for performing symmetry conditions.

Guillaume
